autoload/rainbow_csv.vim

```diff
@@ -835,25 +834,35 @@ func! rainbow_csv#csv_align()
         echoerr 'RainbowAlign not available for "rfc_csv" filetypes, consider using "csv" instead'
         return
     endif
-    let lastLineNo = line("$")
+
+    if (l:firstline == 1 && l:lastline == line("$") && (&ft == 'markdown' || &ft == 'rmd'))
```

Comment:
Please move this check higher, see my other comment.

Reply:
done